src/Fmt_ast.ml

```diff
@@ -352,6 +352,16 @@ let wrap_record c =
   if c.conf.space_around_collection_expressions then wrap "{ " "@ }"
   else wrap_fits_breaks c.conf "{" "}"
 
+let wrap_tuple ~parens ~no_parens_if_break c =
+  let wrap_fits_breaks_if _c cnd pre suf k fs =
```

Comment:
This should probably move to [Fmt]

Reply:
Also, I wonder why code for tuple is so different from list, array, record  ..

Reply:
I'm not sure this function would fit in Fmt, for now Fmt only contains generic functions that are not designed for any particular AST node.
If we want to extract wrap_tuple, wrap_record and so on, maybe we should create a file Fmt_ast_helper to gather the AST specific functions.